app/src/main/java/com/example/paritosh/stormy/WeatherDataProvider.java

```diff
@@ -55,6 +58,30 @@ public void onResponse(@NonNull Call call, Response response) throws IOException
 
     }
 
+    private ArrayList<Hourly> hourlyDetails(String jsonData) throws JSONException {
+
+        JSONObject jsonObject = new JSONObject(jsonData);
+
+        Hourly.locationLabel = jsonObject.getString("timezone");
```

Comment:
You shouldn't use static variables for this purpose.
String timezone = jsonObject.getString("timezone");